odl/solvers/advanced/proximal_operators.py

```diff
@@ -143,26 +251,28 @@ def make_prox(tau):
 
         return IdentityOperator(space)
 
-    return make_prox
+    return make_identity
 
 
-def proximal_nonnegativity(space):
-    """Function to create the proximal operator of G(x) = ind(x > 0).
+def proximal_clamp(space, lower=None, upper=None):
```

Comment:
You're citing "a legacy NVIDIA toolkit no longer under active
development or support"? :-)

On 2016-05-21 22:34, Jonas Adler wrote:
> In odl/solvers/advanced/proximal_operators.py
> <https://github.com/odlgroup/odl/pull/401#discussion_r64139910>:
> 
>>  
>>  
>> -def proximal_nonnegativity(space):
>> -    """Function to create the proximal operator of G(x) = ind(x > 0).
>> +def proximal_clamp(space, lower=None, upper=None):
> 
> Clamp is the standard word in programming, see e.g.
> 
> http://http.developer.nvidia.com/Cg/clamp.html
> 
> With that said, "clamping" is the operation performed, not the
> underlying constraint/function. We have named all others after this so
> i'll go for your |proximal_box_constraint| here.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/odlgroup/odl/pull/401/files/5a721d1d1325be17940d8eeb33e35bdd4879ad8a#r64139910>
> 

-- 

Dr Holger Kohr
KTH - Royal Institute of Technology
Dpt of Mathematics
100 44 Stockholm
Tel +46 8 790-6652
Mobil +46 72 0119332